app/views/family_mailer/summary.html.erb

```diff
@@ -1,9 +1,13 @@
 <h2 class="greeting"><%= t('precheck.status.show.greeting') %></h2>
 
 <p>
-  <% if @family.approved? %>
-    <%= t('.precheck_confirmed_html', conference: UserVariable[:conference_id]) %>
-  <% else %>
+  <% if @family.approved? 
+    if @family.anyone_registered_for_legacy_conferences %>
```

Comment:
You didn't rename this and all the other places the function is called.

Reply:
well, no comment...